### DISCUSSION TITLE
Default language on public view & problem setting dark mode

### DISCUSSION THREAD

**Discussion by JuSchi89** [mealie-recipes/mealie#3898] [Q&A] [answered]
Hey there,

is it possible to set a default language for the public view? If I login the language is right but not on my public view.
Besides this I dont get how to set dark mode. I use the env-variable THEME_DARK_PRIMARY=#E58325 but no change. What am I doing wrong? 

best regards!

**Comment by boc-the-git** [comment]
> is it possible to set a default language for the public view?

From a quick skim, it doesn't appear to be. We'd accept a pull request that adds that, via an environment variable.

> Besides this I dont get how to set dark mode. I use the env-variable THEME_DARK_PRIMARY=#E58325 but no change. What am I doing wrong?

Dark mode is triggered by the button in the bottom left corner. The environment variables don't need touched.

![image](https://github.com/user-attachments/assets/77e34409-b3b5-4c72-a9fc-1677b2d65d78)

**Reply by JuSchi89** [reply]
Oh ok, I see. Yes I know that I can change it like that, but I thought there would be a way to set theme and language in general as default for the public view. It doesn't even need to be the public view only. Setting it for the whole environment also for all new users. And when users want to change it in their own profile later they can do so.

**Reply by michael-genson** [reply]
Theme (dark vs light mode) defaults to the user's system settings

**Reply by JuSchi89** [reply, marked as answer]
Still it would be nice to set any defaults for the theme, no matter what the system setting is. And of course the language. You might also pick the system/browser language here. Simpler I guess would be to just set it as an env variable. Just an overall setting for theme and language which can be adjusted later in the profiles, if needed. Anyway, thanks for the info. For now that's what is there.
